styles/desktop.css

```diff
@@ -0,0 +1,345 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap');
+
+body {
```

Comment:
- Kindly make sure to use @media queries to make your website responsive, making sure your design doesn't break at any point between 375px and 1440 of width :recycle: , while also sticking to the figma design.

![rawcdn githack com_Paulinelali_Personal-portfolio-microverse_33c01a4de90f752a052da691a427141da1c7ed6b_public_index html](https://user-images.githubusercontent.com/97834160/230581249-6d9e70c6-7008-43c4-96df-0836670601a5.png)

Reply:
Hi @GedeonTS 
,
Thank you for your feedback😎

This specific branch you are reviewing is the desktop version and not the mobile version so I don't need to consider breaking points of 375px and 1440px of width. Kindly note that the @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap') is not for media query but for the font(Poppins) used in the Figma design. Please review again as that will really help me to understand what I might be missing out on.

Thank you👏

Happy review☺

Reply:
Hi @Paulinelali, I know the purpose of @import, kindly note that I highlighted that point because it was the top of the file were you wrote the desktop styles, in these purpose you can put there a media query for desktop version 🤓 , because as per the project requirements, when you make your desktop version you should make sure the mobile version is still working properly.

Reply:
I have different style file for my mobile version